src/api/controller/api/run.js

```diff
@@ -48,6 +48,15 @@ export const runRoutine = async (ctx, routineCalled) => {
         host: ctx.host,
         hostname: ctx.hostname,
     };
+    const fields = ctx.path
```

Comment:
Manually parse the request URL isn't recommended.
It's better to pass all the params from the router IMHO

```js
export const runRoutine = async (ctx, routineCalled, field1, field2) => {
    // ...
    const fields = [routineCalled, field1, field].filter(x => x);
}
```

Reply:
Yes, you're right. 
However, some older routines use ctx.path to detect the concerned fields.
I'm afraid to break something, I propose to change nothing to ensure backward compatibility

Reply:
oh... I understood, it's a new variable so ctx.path is still accessible... i'll patch